### PR TITLE
iss1458: added sanity checker for the method that detects point cut t…

### PR DESCRIPTION
**Pull request by dmgcodevil**
…ype by method annotations

### CODE REVIEW

File: hystrix-contrib/hystrix-javanica/src/main/java/com/netflix/hystrix/contrib/javanica/utils/AopUtils.java [hunk 1]
```diff
@@ -123,4 +125,76 @@ public static Method getDeclaredMethod(Class<?> type, String methodName, Class<?
         return Optional.absent();
     }
 
+    public static String getMethodInfo(Method m) {
```
Comment (matterche) [inline hystrix-contrib/hystrix-javanica/src/main/java/com/netflix/hystrix/contrib/javanica/utils/AopUtils.java]: May it make sense to add some information about the declaring class of the method?

Comment (dmgcodevil) [inline hystrix-contrib/hystrix-javanica/src/main/java/com/netflix/hystrix/contrib/javanica/utils/AopUtils.java]: good point, will add

Comment (dmgcodevil) [inline hystrix-contrib/hystrix-javanica/src/main/java/com/netflix/hystrix/contrib/javanica/utils/AopUtils.java]: done

Comment (matterche) [inline hystrix-contrib/hystrix-javanica/src/main/java/com/netflix/hystrix/contrib/javanica/utils/AopUtils.java]: Nice, thanks. For me this looks good.

Comment (dmgcodevil) [inline hystrix-contrib/hystrix-javanica/src/main/java/com/netflix/hystrix/contrib/javanica/utils/AopUtils.java]: @mattrjacobs merge ?

**Pull request merged**